docs/gene_sets.md

```diff
@@ -0,0 +1,103 @@
+## Overview
+
+Users may view gene sets along with their dataset and access gene sets created in pip installed cellxgene by following the instructions below.
+
+The order of the gene symbols in each gene set is maintained in the database. 
```

Comment:
what is `the database`?  Put another way, what is it that this sentence is trying to communicate?